docs/bnb-smart-chain/developers/node_operators/fast_node.md

```diff
@@ -84,3 +84,7 @@ EnableMsgEvents = false
 ## start a fast node
 ./geth --tries-verify-mode full --config ./config.toml --datadir ./node  --cache 8000 --rpc.allow-unprotected-txs --history.transactions 0
 ```
+
+
+### Additional Fast Node Option (Erigon)
+Refer to the Erigon documentation for running a fast node using Erigon.
```

Comment:
The "Erigon fast node" is different from the "fast node" in this doc. We don't need to add this part, just remove it.